### ISSUE TITLE
ensure of resource `puppetserver puppetconf trusted external command` should be configurable

### ISSUE THREAD

**Issue by MartyEwings** [puppetlabs/puppetlabs-servicenow_cmdb_integration#57]
https://github.com/puppetlabs/puppetlabs-servicenow_cmdb_integration/blob/7de591b3a8eefbb4a502943ea968e06582c0b661/manifests/init.pp#L107

There is no uninstall available for the module, if there is an error with the integration, and servicenow.rb does not execute catalogs fail to apply.

The resource puppetserver puppetconf trusted external command should be able to be set to "ensure absent"

**Comment by albatrossflavour** [comment]
Yes, please!  Will make it so much easier to do demo environments if we don't have to manually unpick the snow stuff.

**Comment by dbr787** [comment]
https://github.com/puppetlabs/puppetlabs-servicenow_cmdb_integration/blob/7de591b3a8eefbb4a502943ea968e06582c0b661/manifests/init.pp#L110

In addition to just `present` or `absent`, the `value` needs to be configurable too...
In the scenario where an environment has more than one trusted external command script that needs to be executed (not just servicenow.rb), we need to specify a directory, in which all scripts will be run to generate trusted external facts.
